yapapi/services/service_runner.py

```diff
@@ -44,7 +45,7 @@ class ControlSignal(enum.Enum):
 
 
 class ServiceRunner(AsyncContextManager):
-    def __init__(self, job: "Job"):
+    def __init__(self, job: "yapapi.engine.Job"):
```

Comment:
Hmm, we already have

```
if TYPE_CHECKING:
    from yapapi.engine import Job
```

--> Why is this change necessary?
--> Also: now we don't use the imported `"Job"` -> if this change is indeed needed, please remove the redundant import, as we have `F401 'yapapi.engine.Job' imported but unused` now.

Reply:
> Hmm, we already have
> --> Why is this change necessary? 

because I wanted to include `ServiceRunner` in the API docs but afterwards decided against and added an issue instead

> --> Also: now we don't use the imported `"Job"` -> if this change is indeed needed, please remove the redundant import, as we have `F401 'yapapi.engine.Job' imported but unused` now.

okay, I'll revert the change for now